test/e2e/tests/signature-request.spec.js

```diff
@@ -88,18 +165,31 @@ describe('Sign Typed Data V4 Signature Request', function () {
   });
 });
 
+async function verifyAndAssertSignTypedDataV3(driver) {
```

Comment:
Is it the standard to keep all the helper functions at the top of the file before the tests themselves?

Reply:
I found one example at the end of the page so I moved the other at the end as well.